Backpack-SwiftUI/AppSearchModal/Classes/BPKAppSearchModal.swift

```diff
@@ -76,6 +75,19 @@ public struct BPKAppSearchModal: View {
         .background(.surfaceDefaultColor)
     }
     
+    private var searchInputSummary: some View {
+        BPKSearchInputSummary(
+            placeholder: inputHint,
+            clearAction: .init(accessibilityLabel: clearAction?.accessibilityLabel ?? "", action: {
```

Comment:
it doesn't seem like this should be an optional property? if not, then the `clearAction` property should be non optional